app/models/like.rb

```diff
@@ -0,0 +1,14 @@
+class Like < ApplicationRecord
+  # belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  # def self.update_likes_counter(post_id)
+  #   post = Post.find(post_id)
+  #   post.update(likes_counter: post.likes.count)
+  # end
```

Comment:
- [ ] Comments render our codebase messy. Let's avoid pushing them to our repo :+1: